README.md

```diff
@@ -50,9 +50,34 @@ do you want to permanently mark this value as trusted (y/N)? y
 
 ```
 
-If you see the prompts above, terminate the command and first add IOG's binary caches to your Nix configuration as described [here](https://input-output-hk.github.io/haskell.nix/tutorials/getting-started.html#setting-up-the-binary-cache).
+Accepting these prompts will set the required configuration values for you. Marking them as trusted will ensure that they are used for future `nix` invocations in this repository.
```

Comment:
@ngua accepting the prompts did not work for me. I had the same issue described here: https://github.com/NixOS/nix/issues/6672

Was my problem the thing to do with trusted users below?

Reply:
Possibly, if you aren't a trusted user